### ISSUE TITLE
Repeated content

### ISSUE THREAD

**Issue by Aurel300** [skial/haxe.io#642]
First of all, thank you very much for maintaining the round-ups for such a long time!

I have enjoyed reading through the round-ups but I must say recently (-ish) it's been less interesting. What makes it less attractive for me is the presence of a lot of content that is repeated from previous weeks. I mentioned this [before](https://github.com/skial/haxe.io/issues/605#issuecomment-479233855), but that was just a comment in passing.

Some ideas/suggestions:

 - Could the "In case you missed it" sections be visually collapsed by default?
 - Could there be some system to determine how long a "long-term" post should stay up? E.g. job postings could show for 10 (maybe fewer?) weeks by default, unless the related issue is closed on GitHub earlier. Articles, tweets, game releases could probably have a shorter time to live, perhaps suggested by the author in the round-up issue?
 - Could the postings be organised with references to the round-up they were originally suggested? That would also make it easier to automate the "time to live".

The latter two points seem like more major changes, though ultimately might lead to less work in maintaining the round-ups?

**Comment by Gama11** [comment]
I agree, I've been merely skimming over the roundups recently because it always feels like I've already seen most of the content in previous roundups. I can't just read the parts that are new either, because there's no clear indicator of what's new and what's old.

**Comment by skial** [comment]
Thanks @Aurel300 & @Gama11, all fair points.

> Could the "In case you missed it" sections be visually collapsed by default?

I think this is a great idea. 👍

> Could there be some system to determine how long a "long-term" post should stay up? E.g. job postings could show for 10 (maybe fewer?) weeks by default, unless the related issue is closed on GitHub earlier. Articles, tweets, game releases could probably have a shorter time to live, perhaps suggested by the author in the round-up issue?

This has been a recurring issue in the back of my mind. I'm more inclined to limit the majority of job posts to 2-4 weeks, other content 2 weeks tops...?

> Could the postings be organised with references to the round-up they were originally suggested? That would also make it easier to automate the "time to live".

I like this idea but I need to look into how to do it efficiently.

> I can't just read the parts that are new either, because there's no clear indicator of what's new and what's old.

_Generally_, content before any _"in case you missed it"_ sections should be new the week of the roundup release.

**Comment by tobil4sk** [comment]
One thing that's stuck out to me for a while is that some of the "in case you missed it" sections are collapsible, whereas others are just regular headings. Is there a specific reason for this? Or is it an oversight? It looks like in older roundups all of these sections were collapsible as expected. For example: https://haxe.io/roundups/501/

**Comment by skial** [comment]
It looks like something broke and I've just completely missed it 🤦‍♂️

**Comment by tobil4sk** [comment]
They seem to use different heading levels:
https://github.com/skial/haxe.io/blob/a64c7edfaac786c6c388473fcd2893cf489c992e/src/roundups/648.md?plain=1#L17
vs
https://github.com/skial/haxe.io/blob/a64c7edfaac786c6c388473fcd2893cf489c992e/src/roundups/648.md?plain=1#L58

**Comment by tobil4sk** [comment]
Nice, all good now, thanks!

**Comment by tobil4sk** [comment]
Actually, one more thing 😅 this one seems not to be collapsible either:

https://github.com/skial/haxe.io/blob/52b49ac9e3e14c49256f606fc75598f8c9901895/src/roundups/649.md?plain=1#L74

**Comment by skial** [comment]
> Actually, one more thing 😅 this one seems not to be collapsible either:
> 
> https://github.com/skial/haxe.io/blob/52b49ac9e3e14c49256f606fc75598f8c9901895/src/roundups/649.md?plain=1#L74

Thankfully that one is on purpose. :+1:
